src/ape_ethereum/ecosystem.py

```diff
@@ -90,7 +182,10 @@ def decode(cls, data: dict) -> ReceiptAPI:
 
 
 class Ethereum(EcosystemAPI):
-    transaction_class = Transaction
+    transaction_class_map = {
+        "0x0": StaticFeeTransaction,
```

Comment:
Does anyone use type `0x1` ?
I think we may need an intermediary tx class to support it but I am not sure why it would be needed

Reply:
Yes, static fee transactions are still used by people on mainnet, and other chains that haven't migrated yet. also, combing through history